include/dao/dao.js

```diff
@@ -114,6 +117,48 @@ module.exports = function DAOModule(pb) {
         this.loadByValues(where, collection, opts, cb);
     };
 
+    DAO.prototype.loadByValueAvailableToSite = function(key, val, site, collection, opts, cb) {
```

Comment:
Passing the site to every function seems very heavy handed.  The DAO constructor should except an options argument that is an object.  The object should not be required.  If object should support a ```site``` property that can be set for the instance of DAO.  This will prevent the extra parameters from being passed around.